policy-test/src/lib.rs

```diff
@@ -168,6 +168,14 @@ pub async fn await_route_status(
     .inner
 }
 
+// Wait for the endpoints controller to populate the Endpoints resource.
+pub fn endpoints_ready(obj: Option<&k8s::Endpoints>) -> bool {
+    if let Some(ep) = obj {
+        return ep.subsets.iter().flatten().count() > 0;
+    }
+    false
```

Comment:
nit, take it or leave it: could also be represented as 
```suggestion
   obj.into_iter().flat_map(|ep| ep.subsets.iter().flatten()).count() > 0
```
although :woman_shrugging: if that's any clearer,